manager/src/main/java/org/openremote/manager/mqtt/ActiveMQORSecurityManager.java

```diff
@@ -98,13 +104,58 @@ protected Subject getAuthenticatedSubject(String user,
         String realm = null;
         ClassLoader currentLoader = Thread.currentThread().getContextClassLoader();
         ClassLoader thisLoader = this.getClass().getClassLoader();
-
+        X509Certificate[] certs = getCertsFromConnection(remotingConnection);
         if (user != null) {
             String[] realmAndUsername = user.split(":");
             if (realmAndUsername.length == 2) {
                 realm = realmAndUsername[0];
                 user = realmAndUsername[1];
             }
+        }else if (certs != null && certs.length > 0) {
+
+            X509Certificate leaf = certs[0];
+            String dn = leaf.getSubjectX500Principal().getName();
+            try {
+                // Check for Client Authentication EKU (This denotes that this is the actual client certificate to enforce)
+                if(!leaf.getExtendedKeyUsage().contains("1.3.6.1.5.5.7.3.2")) {
+                    // TODO: Not sure about what extent to which we would like to enforce this.
```

Comment:
LOG and fail as it suggests misconfiguration and we shouldn't make exceptions to things not following best practices